app/controllers/specialist_documents_controller.rb

```diff
@@ -8,7 +8,7 @@ class SpecialistDocumentsController < ApplicationController
   def show
     document = content_store.content_item(base_path)
 
-    if document['format'] != 'gone'
+    if document.schema_name != 'gone'
```

Comment:
This code does not look like it relates to schemas or validation, so why use `schema_name` instead of `document_type`?